.github/workflows/tests.yml

```diff
@@ -21,11 +21,37 @@ jobs:
         run: ./tests/install-yq.sh
       - name: Verify CODEOWNERs
         run: ./tests/code-owner-check.sh
-      - name: Static File checks
-        run: ./tests/static-file-checks.sh
-      - name: Kustomize Testsv2
-        run: ./tests/test-kustomize-v2.sh
+      - name: Image Policy Tests
+        run: ./tests/image-policy-exists.sh
       - name: Lint Tests
         run: ./tests/lint-yaml.sh
       - name: Fluxv2 Image Automation
-        run: ./tests/flux-v2-image-automation.sh
\ No newline at end of file
+        run: ./tests/flux-v2-image-automation.sh
+  dryrun_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        environment: [sbox, sbox-intsvc, preview, ptl-intsvc, ithc, perftest, demo, aat, prod]
+        cluster: ["00", "01"]
+        exclude:
+          - environment: sbox-intsvc
+            cluster: "01"
+          - environment: ptl-intsvc
+            cluster: "01"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install yq
+        run: ./tests/install-yq.sh
+      - name: Install Flux CLI
+        run: |
+          FLUX_VERSION=$(yq eval 'select(.kind=="Namespace") | .metadata.labels."app.kubernetes.io/version"' apps/flux-system/base/gotk-components.yaml | sed 's/v//')
```

Comment:
Picking the flux version from cluster as the latest one seems to have some breaking changes